x/profiles/types/events.go

```diff
@@ -1,11 +1,18 @@
 package types
 
 const (
-	EventTypeProfileSaved   = "profile_saved"
-	EventTypeProfileDeleted = "profile_deleted"
+	EventTypeProfileSaved            = "profile_saved"
+	EventTypeProfileDeleted          = "profile_deleted"
+	EventTypeDTagTransferRequest     = "dtag_transfer_request"
+	EventTypeDTagTransferReqAccepted = "accepted_transfer_request"
 
 	// Profile attributes
 	AttributeProfileDtag         = "profile_dtag"
 	AttributeProfileCreator      = "profile_creator"
 	AttributeProfileCreationTime = "profile_creation_time"
+
+	// DTag trade attributes
+	AttributeCurrentOwner  = "current_owner"
+	AttributeReceivingUser = "receiving_user"
+	AttributeNewDTag       = "new_DTag"
```

Comment:
I think it's better to make this all lowercase

```suggestion
	AttributeNewDTag       = "new_dtag"
```